src/control/dmg/pool.go

```diff
@@ -58,6 +59,49 @@ type CreatePoolCmd struct {
 	Sys        string `short:"S" long:"sys" default:"daos_server" description:"DAOS system that pool is to be a part of"`
 }
 
+// Execute is run when CreatePoolCmd subcommand is activated
+func (c *CreatePoolCmd) Execute(args []string) error {
+	// broadcast == false to connect to mgmt svc access point
+	if err := appSetup(false); err != nil {
+		return err
+	}
+
+	if err := createPool(
+		c.ScmSize, c.NVMeSize, c.RankList, c.NumSvcReps,
+		c.GroupName, c.UserName, c.Sys, c.ACLFile); err != nil {
+
+		return err
+	}
+
+	// exit immediately to avoid continuation of main
+	os.Exit(0)
```

Comment:
I have some concerns about having to copy this boilerplate code for every command. I'll create a separate ticket to address that, though.

Reply:
thanks

Reply:
FYI: https://github.com/daos-stack/daos/pull/673